apps/dashboard/src/app/resources/resource-list.component.ts

```diff
@@ -69,6 +74,8 @@ export class ResourceListComponent implements OnInit, OnDestroy {
   unsubscribeAll = new Subject<void>();
   refreshing = true;
 
+  ////////////////////////////////////////////////////////////////////////////////////////////////////////
+  // FIXME: those options are not used anywhere, can we remove them or is there a reason to keep them here?
```

Comment:
I think the resource list page is supposed to have a `status` and `type` filters, but haven't been implemented in the back yet. I guess it can be deleted.

Reply:
ok, let's keep them for now, I'll ask Ramon next week (or whenever they'll have finished to fix what they're trying to fix lately) about those filters and we'll remove them depending on the answer.